src/devices/Hx711/samples/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// See https://aka.ms/new-console-template for more information
+using System.Threading;
+using System;
+using System.Device.Gpio;
+using Iot.Device.Hx711;
+using UnitsNet;
+
+int pinDout = 23;
+int pinPD_Sck = 24;

```

Comment:
tiny one here: pinPdSck (and I guess same in the readme)

Reply:
Ok, I fix it (and in readme too).